satellite/metainfo/metainfo.go

```diff
@@ -188,6 +189,34 @@ func (endpoint *Endpoint) CreateSegment(ctx context.Context, req *pb.SegmentWrit
 	return &pb.SegmentWriteResponse{AddressedLimits: addressedLimits, RootPieceId: rootPieceID}, nil
 }
 
+func (endpoint *Endpoint) getProjectStorageTotals(ctx context.Context, projectID uuid.UUID) (int64, int64, error) {
+	lastCountInline, lastCountRemote, err := endpoint.accountingDB.ProjectStorageTotals(ctx, projectID)
+	if err != nil {
+		return 0, 0, err
+	}
+	rtInline, rtRemote, err := endpoint.liveAccounting.GetProjectStorageUsage(ctx, projectID)
+	if err != nil {
+		return 0, 0, err
+	}
+	return lastCountInline + rtInline, lastCountRemote + rtRemote, nil
+}
+
+func calculateSpaceUsed(ptr *pb.Pointer) (inlineSpace, remoteSpace int64) {
+	inline := ptr.GetInlineSegment()
+	if inline != nil {
+		return int64(len(inline)), 0
+	}
+	segmentSize := ptr.GetSegmentSize()
+	remote := ptr.GetRemote()
+	if remote == nil {
+		return 0, 0
+	}
+	minReq := remote.GetRedundancy().GetMinReq()
+	pieceSize := segmentSize / int64(minReq)
+	pieces := remote.GetRemotePieces()
+	return 0, pieceSize * int64(len(pieces))
```

Comment:
`pieceSize := segmentSize / int64(minreq)` is an inherent part of our system; pieces end up being this big as a consequence of the reed-solomon encoding. See pkg/accounting/tally/tally.go:180, pkg/eestream/encode.go:274, pkg/storage/ec/client.go:346, etc.

`pieceSize * len(pieces)` is because the post-expansion-factor space used is the size of each piece times the number of pieces we have stored.